### ISSUE TITLE
Dealing with flexible efficiencies

### ISSUE THREAD

**Issue by wouterterlouw** [quintel/atlas#26]
This ticket describe the functionality required to model the Industry for both The Netherlands and Germany.
### The Problem

_1) industry_transformation_generic_coal_

Part of the coal use in the Industry is used for transformation purposes. This is not counted in the final demand of coal, but result in primary energy requirements of coal. When the metal industry is switched on, this is modeled almost correctly for NL. However, when the metal industry will be switched off, this coal requirement need to be accounted for in the `industry_transformation_generic_coal` converter by setting the `output.coal` conversion (coal to final demand) and the `output.loss` conversion (coal that to be modeled).

This efficiency is dependent on 1) whether or not the metal industry is specified, 2) the quality of the metal analysis.

See also https://github.com/quintel/rdr/issues/106 for the research background.

_2) Blastfurnaces and cokesovens_

The efficiencies of blastfurnaces and cokesovens are quite dependent on local circumstances. Therefore, e.g. `input.coal`, `input.cokes` and `output.coal_gas` are calculated in the analysis and currently set in the respective converters. However, it is most likely that the efficiencies for Germany will differ from the efficiencies of The Netherlands. Also efficiencies between years can differ.

See also https://github.com/quintel/rdr/issues/107 for the research background.
### Potential solution

A potential solution for the technical part of this problem (as @antw suggest in https://github.com/quintel/rdr/issues/64#issuecomment-21788427) is to query the efficiencies based on a table that is generated in a analysis that is output from a analysis, e.g. in case of `industry_transformation_generic_coal` for NL 2011:

```
~ output.coal = EFF(…, output.coal)
~ output.loss = EFF(…, output.loss)
```
